components/app/explore/explore-dataset-filters/explore-dataset-filters-actions.js

```diff
@@ -0,0 +1,21 @@
+import { createAction, createThunkAction } from 'redux-actions';
+
+import FiltersService from 'services/FiltersService';
+
+export const setDataFilters = createAction('explore-dataset-filters/setDataFilters');
+export const setFilter = createAction('explore-dataset-filters/setFilter');
```

Comment:
Loading an Error actions `setFilterLoading` and `setFilterError`

Reply:
I miss the default state. I recommend you to start always by defining the default state because it will give a good shape of what you are going to achieve.

Reply:
Remember that you can't add logic inside the `xxx-component.js`